packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -71,6 +71,19 @@ ${JSON.stringify(actualEvents)}`);
 expected:
 ${JSON.stringify(expectedEvents)}
 
+actual:
+${JSON.stringify(actualEvents)}`);
+            }
+    }
+
+    /** Asserts that matchAnyEvent is false for the given events, and prints the actual/expected output if not */
+    assertMatchNone(disallowedEvents: Omit<ITelemetryBaseEvent, "category">[], message?: string) {
```

Comment:
I used a local version of this in the `gcInactiveNodes.spec.ts`. I will replace it with this one later :-).